modules/core/src/main/java/com/smartbear/readyapi4j/ExecutionListenerAdapter.java

```diff
@@ -0,0 +1,31 @@
+package com.smartbear.readyapi4j;
+
+import com.smartbear.readyapi.client.model.ProjectResultReport;
+import com.smartbear.readyapi4j.execution.Execution;
+
+public abstract class ExecutionListenerAdapter implements ExecutionListener {
```

Comment:
I'm not sure that we should use default methods for this. It's tempting (and cool!) but default methods were not introduced with this use case in mind (it was more about letting interfaces evolve without breaking backward compatibility), so it may be confusing. Adapter classes are generally non-abstract though - arguably a bad thing, but it is the convention - so remove abstract.